### PR TITLE
Document configurable HTTP headers

### LINKED ISSUE #2910: Document configurable HTTP headers
PR: https://github.com/influxdata/influxdb/pull/18667

Users can now specify specific http headers for influxdb to return.

These pages should be updated:

- https://github.com/influxdata/docs.influxdata.com/blob/master/content/influxdb/v1.8/administration/config.md#http-endpoints-settings
- https://github.com/influxdata/docs.influxdata.com/blob/master/content/influxdb/v1.8/administration/https_setup.md and/or https://github.com/influxdata/docs.influxdata.com/blob/master/content/influxdb/v1.8/administration/security.md, noting that this can be used to help comply with security policies regarding headers.

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2910

### CODE REVIEW

File: content/influxdb/v1.8/administration/config.md [hunk 1]
```diff
@@ -682,6 +682,19 @@ This setting has no effect if [`auth-enabled`](#auth-enabled-false) is set to `f
 
 Environment variable: `INFLUXDB_HTTP_PING_AUTH_ENABLED`
 
+#### `http-headers`
+
+User-supplied [HTTP response headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers).
+Configure this section to return [security headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers#Security)
```
Comment (kelseiv) [inline content/influxdb/v1.8/administration/config.md:688]: @pierwill could folks also configure headers other than security here? For example, `content_encoding = "gzip"`?

Comment (pierwill) [reply content/influxdb/v1.8/administration/config.md:688]: Definitely. Security headers is an important use case we want to highlight here.

Comment (pierwill) [reply content/influxdb/v1.8/administration/config.md:688]: Maybe add a sentence in the middle here: "Users can configure any header..."

Comment (timhallinflux) [reply content/influxdb/v1.8/administration/config.md:688]: I think the question is whether InfluxDB can respond to the header.... in the case of gzip that is a kind of behavior change....

File: content/influxdb/v1.8/administration/security.md [hunk 2]
```diff
@@ -36,6 +36,14 @@ HTTPS can also verify the authenticity of the InfluxDB server to connecting clie
 Resources:
 [Enabling HTTPS](/influxdb/v1.8/administration/https_setup/)
 
+## Configure security headers
+
+HTTP headers allow servers and clients to pass additional information along with requests.
+Certain headers help [enforce security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers#Security) properties.
+
+Resources:
+[Configure HTTP headers](http://localhost:1314/influxdb/v1.8/administration/config/#http-headers)
```
Comment (timhallinflux) [inline content/influxdb/v1.8/administration/security.md:45]: I think this URL looks wrong...  can you review/address?

Comment (pierwill) [reply content/influxdb/v1.8/administration/security.md]: Yep, fixed!

**Pull request merged**